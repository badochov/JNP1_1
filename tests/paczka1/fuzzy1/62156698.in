Ny5IcfAReEg S672 9,2
spPBrTaMWm S795 52,2
spPBrTaMWm S795 39,2
?S795
S795 S795 26,8
S795 S337 71,4
ddiD S337 57,9
S795 S337 17,1
?S337

Ny5IcfAReEg S672 28,2
spPBrTaMWm S795 138,0
spPBrTaMWm S335 92,2
?
ddiD S337 64,8
ddiD S335 70,3
spPBrTaMWm S335 51,3
?S795
4LFUvLt S335 97,0
spPBrTaMWm S795 49,5
PnH S337 88,1
?S795
PnH S337 415,0
ddiD S335 190,5
Ny5IcfAReEg S795 13,7
Ny5IcfAReEg S337 50,4
Ny5IcfAReEg S335 3,9
spPBrTaMWm S672 45,2
jns S335 80,9
?S335
4LFUvLt S335 286,2
Ny5IcfAReEg S335 29,2
Ny5IcfAReEg S335 128,5
?
spPBrTaMWm S335 20,8
spPBrTaMWm S335 17,5
PnH S795 45,7
jns S335 226,5
S795 S337 91,5
