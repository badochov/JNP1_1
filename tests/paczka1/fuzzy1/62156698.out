S795 13,0
S337 54,3
Ny5IcfAReEg S 19,0
S795 S 54,3
spPBrTaMWm S 13,0
S337 54,3
S672 19,0
S795 13,0
S795 S 54,3
S795 13,0
S795 S 54,3
S795 13,0
S335 161,1
4LFUvLt S 189,2
Ny5IcfAReEg S 44,3
PnH S 326,9
S795 S 54,3
ddiD S 127,1
spPBrTaMWm S 53,9
S335 375,6
S337 388,1
S672 19,0
S795 13,0
